package.json

```diff
@@ -59,6 +59,7 @@
     "eslint-plugin-import": "2.26.0",
     "eslint-plugin-mocha": "10.1.0",
     "eslint-plugin-prettier": "4.2.1",
+    "eslint-plugin-simple-import-sort": "^10.0.0",
```

Comment:
I think you changed for the other plugin, so let's remove this from dependencies